e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/sdk-iframe-embedding-setup.cy.spec.ts

```diff
@@ -0,0 +1,62 @@
+const { H } = cy;
+
+describe("scenarios > embedding > sdk iframe embedding setup flow", () => {
+  beforeEach(() => {
+    H.restore();
+    cy.signInAsAdmin();
+    H.setTokenFeatures("all");
+
+    cy.intercept("GET", "/api/dashboard/**").as("dashboard");
+  });
+
+  it("shows dashboard experience by default", () => {
+    cy.visit("/embed/new");
+    cy.wait("@dashboard");
+
+    const iframe = getPreviewIframe();
+    iframe.within(() => {
+      cy.log("dashboard title is visible");
+      cy.findByText("Person overview").should("be.visible");
+
+      cy.log("dashboard card is visible");
+      cy.findByText("Person detail").should("be.visible");
+    });
```

Comment:
Test instance does not have a sample database, so it defaults to `dashboardId: 1`, activity log.

In the next PR EMB-508 where we introduce selecting dashboards and questions for new iframe embedding from recents and search, the default dashboard will be your last viewed dashboard instead of `1`.

![CleanShot 2568-06-21 at 02 21 44@2x](https://github.com/user-attachments/assets/c7664c78-08c4-4f1e-84aa-7db5e1148f0a)